support/Pipelines/Bbh/FindHorizon.py

```diff
@@ -78,17 +96,24 @@ def find_horizon(
         'spectre.SphericalHarmonics.Strahlkorper'.
       fast_flow: Optional. FastFlow object that controls the horizon finder.
         If not specified, a FastFlow object with default parameters is used.
-      output: Optional. H5 output file where the horizon Ylm coefficients will
-        be written. Can be a new or existing file. Requires either
-        'output_coeffs_subfile' or 'output_coords_subfile' is also specified,
-        or both.
-      output_coeffs_subfile: Optional. Name of the subfile in the H5 output file
-        where the horizon Ylm coefficients will be written.
-        These can be used to reconstruct the horizon, e.g. to initialize
-        excisions in domains.
-      output_coords_subfile: Optional. Name of the subfile in the H5 output file
-        where the horizon coordinates will be written.
+      output_surfaces_file: Optional. H5 output file where the horizon Ylm
+        coefficients will be written. Can be a new or existing file. Requires
+        either 'output_coeffs_subfile' or 'output_coords_subfile' is also
+        specified, or both.
+      output_coeffs_subfile: Optional. Name of the subfile in the
+        'output_surfaces_file' where the horizon Ylm coefficients will be
+        written. These can be used to reconstruct the horizon, e.g. to
+        initialize excisions in domains.
+      output_coords_subfile: Optional. Name of the subfile in the
+        'output_surfaces_file' where the horizon coordinates will be written.
         These can be used for visualization.
+      output_reductions_file: Optional. H5 output file where the reduction
+        quantities on the horizon will be written, e.g. masses and spins.
```

Comment:
I think we should keep all the horizon stuff in the same H5 file. It reduces the number of files on disk and that we have to keep track of. (like forwarding options through pipeline commands)

Reply:
The evolutions write to a separate surfaces file though. Don't you want to be consistent with that? For reading in the data I think you can't assume that both the masses+spins and the shape coeffs are in the same file anyway.

Reply:
I like the fixup you did for the postprocess step just writing things into a single `Horizons.h5` file. In the actual horizon find CLI, having the option for separate files is fine.